roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -34,6 +42,8 @@ def __init__(self, execute_module=None, task_vars=None, tmp=None):
         self._execute_module = execute_module
         self.task_vars = task_vars or {}
         self.tmp = tmp
+        # set True when the check makes a change to the host so it can be reported to the user:
```

Comment:
set to True :+1: 

"report to user" right that can be worded better. All we're doing is marking the task as having made a change so that the sum total of "changed" tasks can be reported at the end of the run. As if anyone cares (but there's always someone). That's all that's "reported" to the user unless they're looking at `-vvv` output.

        # set to True when the check makes a change to the host so the total "changed" count is accurate
￼